core/packages/contracts/foundry.toml

```diff
@@ -0,0 +1,16 @@
+[profile.default]
+solc = '0.8.9'
+via_ir = false
+src = 'contracts'
+out = 'out'
+libs = ["lib","node_modules"]
+test = 'contracts/test'
+fs_permissions = [{ access = "read", path = "./"}]
+allow_paths = ["node_modules/@openzeppelin/contracts"]
+remappings = [
+    'ds-test=lib/ds-test/src/',
+    'forge-std=lib/forge-std/src/',
+    '@openzeppelin=node_modules/@openzeppelin'
+]
+ffi = true
+fuzz_runs = 5000
```

Comment:
Looks like this should be [`runs` under a `[fuzz]`](https://book.getfoundry.sh/reference/config/testing#runs) section. Are we using fuzzing yet? Can remove this if not.